phys2denoise/metrics/multimodal.py

```diff
@@ -23,12 +23,8 @@ def retroicor(
 
     Parameters
     ----------
-    physio : array_like
-        1D array, whether cardiac or respiratory signal.
-        If cardiac, the array is a set of peaks in seconds.
-        If respiratory, the array is the actual respiratory signal.
-    sample_rate : float
-        Physio sample rate, in Hertz.
+    data : Physio_like
+        Object containing the timeseries of the recorded respiratory or cardiac signal
```

Comment:
I would suggest including a bit more information here, especially that the object should also contain associated meta-data.

Reply:
(I think it's worth a discussion where we talk about advantages of forcing some parameters to be in the Physio object vs. explicitly specified before we go too far down this road!)